build_tools/ci/cpu_comparison/run.py

```diff
@@ -669,6 +671,76 @@ def _execute(self, config):
         return True
 
 
+class MatmulTrunci(BaseMatmul):
+    """
+    A test of the form matmul(A,B) + trunci(C) where A:MxK, B:KxM and C:MxM
```

Comment:
Why `MxKxM` and not `MxKxN`? The latter should work as well I would think, so it would be better to provide general matmul support.